### PR TITLE
Clean up events usage

### PR DESCRIPTION
**Pull request by PaulWessel**
Part of the redesign of usage message via #5341.

### CODE REVIEW

File: src/psevents.c [hunk 1]
```diff
@@ -167,79 +167,93 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSEVENTS_CTRL *C) {	/* Deall
...
-	GMT_Message (API, GMT_TIME_NONE, "\t   +p<dt> sets the length of the plateau after event happens [no plateau].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +d<dt> sets the decay-time after the plateau [no decay].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +f<dt> sets the fade-time after the event ends [no fade time].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +l<dt> sets alternative label duration [same as symbol].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-F Specify values for text attributes that apply to all text records:\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +a<angle> specifies the baseline angle for all text [0].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     Use +A to force text-baselines in the -90/+90 range.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +f<fontinfo> sets the size, font, and optionally the text color [%s].\n",
+	GMT_Usage (API, -2, "Record format: lon lat [z] [size] time [length|time2].");
+	GMT_Usage (API, 1, "\n-Ar[<dpu>[c|i]]|s");
```
Comment (meghanrjones) [inline src/psevents.c:185]:
```suggestion
	GMT_Usage (API, 1, "\n-Ar[<dpu][c|i]|s");
```

Comment (meghanrjones) [reply src/psevents.c:185]: Either this needs updating or the rst docs should be changed to agree with GMT_Usage

File: src/psevents.c [hunk 2]
```diff
@@ -167,79 +167,93 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSEVENTS_CTRL *C) {	/* Deall
...
+		"Append <dpu> to convert your line records into dense point records that can be plotted as circles. "
+		"The resampled line will be written to stdout (requires options -R -J, optionally -C). "
+		"The <dpu> must be the same as the intended <dpu> for the movie frames. "
+		"Append i if dpi and c if dpc [Default will consult GMT_LENGTH_UNIT setting, currently %s].", API->GMT->session.unit_name[API->GMT->current.setting.proj_length_unit]);
+	GMT_Usage (API, 3, "s: Read whole segments (lines or polygons) with no time column. n"
+		"Time is set via segment header -T<start>, -T<start>,<end>, or -T<start>,<duration (see -L).");
+	GMT_Usage (API, 1, "\n-C<cpt>");
+	GMT_Usage (API, -2, "Give <cpt> and obtain symbol color via z-value in 3rd data column.");
+	GMT_Usage (API, 1, "\n-D[j|J]<dx>[/<dy>][+v[<pen>]");
+	GMT_Usage (API, -2, "Add <add_x>,<add_y> to the event text origin AFTER projecting with -J [0/0]. "
```
Comment (meghanrjones) [inline src/psevents.c:197]:
```suggestion
	GMT_Usage (API, -2, "Add <dx>,<dy> to the event text origin AFTER projecting with -J [0/0]. "
```

File: src/psevents.c [hunk 3]
```diff
@@ -167,79 +167,93 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSEVENTS_CTRL *C) {	/* Deall
...
+		"Use -Dj to move text origin away from point (direction determined by text's justification). "
+		"Upper case -DJ will shorten diagonal shifts at corners by sqrt(2). "
+		"Append +v[<pen>] to draw line from text to original point.  If <add_y> is not given it equals <add_x>.");
+	GMT_Usage (API, 1, "\n-E[s|t][+o|O<dt>][+r<dt>][+p<dt>][+d<dt>][+f<dt>][+l<dt>]");
```
Comment (meghanrjones) [inline src/psevents.c:201]:
```suggestion
	GMT_Usage (API, 1, "\n-Es|t[+o|O<dt>][+r<dt>][+p<dt>][+d<dt>][+f<dt>][+l<dt>]");
```
EIther this or rst docs need updating

File: src/psevents.c [hunk 4]
```diff
@@ -167,79 +167,93 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSEVENTS_CTRL *C) {	/* Deall
...
+	GMT_Usage (API, 3, "+g Set fill for the text box [no fill].");
+	GMT_Usage (API, 3, "+p Draw outline of the text box [no outline], optionally append pen [%s].", gmt_putpen (API->GMT, &API->GMT->current.setting.map_default_pen));
+	GMT_Usage (API, 3, "+r Select rounded rectangular box shape [straight].");
+	GMT_Usage (API, 3, "+s Plot a shadow behind the text box, optionally append offsets and shade [%gp/%gp/gray50].", GMT_FRAME_CLEARANCE, -GMT_FRAME_CLEARANCE);
+	GMT_Option (API, "K");
+	GMT_Usage (API, 1, "\n-L[t|<length>]");
+	GMT_Usage (API, -2, "Set finite length of events, otherwise we assume they are all infinite. "
+		"If no arg we read lengths from file; append t for reading end times instead. "
+		"If -L0 is given the event only lasts one frame.");
+	GMT_Usage (API, 1, "\n-Mi|s|t<val1>[+c<val2]");
```
Comment (meghanrjones) [inline src/psevents.c:235]: rst docs need updating if <val1> is not optional

**Review by meghanrjones — APPROVED** [review]

File: src/psevents.c (hunk 1 above)
Comment (PaulWessel) [reply src/psevents.c:185]: Fixed. The optional units need to go inside the brackets with the dpu.

File: src/psevents.c (hunk 3 above)
Comment (PaulWessel) [reply src/psevents.c:201]: Fixed the rst.

File: src/psevents.c (hunk 4 above)
Comment (PaulWessel) [reply src/psevents.c:235]: Fixed rst.

**Pull request merged**